client/src/pages/Signup/index.js

```diff
@@ -0,0 +1,226 @@
+  align-items: center;
+`
+const HelperText = styled.p`
+  letter-spacing: 1px;
+  padding: 0;
+  margin: 0 0 0 1em;
+  font-size: 0.8rem;
+  color: ${(props) => (props.error ? 'red' : 'black')};
+`
+const HelperTextWrapper = styled.div`
+  margin: 0 0 0.9em 0;
+  padding: 0;
+
+  width: 80%;
+  text-align: start;
+`
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+const LoginButton = styled.button`
+  width: 80%;
+  border-radius: 25px;
+  padding: 1rem 1.2rem;
+  background: #4933ef;
+  outline: none;
+  color: white;
+  font-size: 1rem;
+  text-transform: uppercase;
+  letter-spacing: 2px;
+  cursor: pointer;
+  box-shadow: 0 3px 5px -1px rgba(0, 0, 0, 0.2),
+    0 6px 10px 0 rgba(0, 0, 0, 0.14), 0 1px 18px 0 rgba(0, 0, 0, 0.12);
+`
+
+const NoAccountWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+const NoAccount = styled.p``
+
+const SignUpLink = styled.button`
+  background: none;
+  display: inline-block;
+  text-decoration: underline;
+  margin: 0;
+  padding: 0;
+  cursor: pointer;
+`
+export default () => {
+  const [email, setEmail] = useState('email...')
+  const [password, setPassword] = useState('')
+  const [name, setName] = useState('Name')
+  const [nameErr, setNameErr] = useState(false)
+  const [pwdErr, setPwdErr] = useState(false)
+  const [emailErr, setEmailErr] = useState(false)
+
+  const validateNameTxt = (text) => (/[^a-zA-Z]/gi.test(text) ? 'error' : '')
+
+  const updateNameText = (value) =>
+    value === 'error' ? setNameErr(true) : setNameErr(false)
+
+  const validateEmailText = (text) => {
+    if (text.includes('@')) return
+    return 'error'
+  }
+
+  const updateEmailText = (value) => {
+    return value === 'error' ? setEmailErr(true) : setEmailErr(false)
+  }
+
+  const validatePwdText = (text) => {
+    if (text.length > 6) return
+    return 'error'
+  }
+
+  const updatePwdText = (value) => {
+    return value === 'error' ? setPwdErr(true) : setPwdErr(false)
+  }
+
+  const handleChange = (e) => {
+    const targetEventName = e.target.name
+    const targetEventValue = e.target.value
+
+    if (targetEventName === 'email') {
+      setEmail(targetEventValue)
+      updateEmailText(validateEmailText(targetEventValue))
+    } else if (targetEventName === 'password') {
+      setPassword(targetEventValue)
+      updatePwdText(validatePwdText(targetEventValue))
+    } else {
+      setName(targetEventValue)
+      updateNameText(validateNameTxt(targetEventValue))
+    }
+  }
+  const handleSubmit = (e) => {
+    e.preventDefault()
+
+    setEmail('')
+    setPassword('')
+  }
+
+  return (
+    <Container>
+      <Form onSubmit={handleSubmit}>
+        <FormTitle>Register Today!</FormTitle>
+        <InputContainer>
+          <Input
+            value={name}
+            type="text"
+            name="name"
+            onChange={handleChange}
+            error={nameErr}
+          />
```

Comment:
Does name input here stands for name + first name ?

Reply:
Good remark. It's now 2 inputs, one for first name and another one for last name